docker/api/Dockerfile

```diff
@@ -19,7 +19,7 @@ ADD docker/api/healthcheck.sh /app/healthcheck.sh
 ADD docker/api/run-gunicorn.sh /app/run-gunicorn.sh
 ADD docker/docker-entrypoint.sh /docker-entrypoint.sh
 
-ENV TESTING_UI="False"
+ENV TESTING_UI="True"
```

Comment:
Please don't change the default. We're using this Dockerfile to run in production (via Kubernetes) and don't want to enable the testing UI in a production environment. You can set the environment variable when you run a Dockerfile locally using the `docker run -e TESTING_UI=True` flag, or alternatively add it to the docker-compose file like so:

```yaml
environment:
  TESTING_UI: "True"
```

Reply:
👍🏽